installer/scripts/tomlparser.rb

```diff
@@ -117,19 +119,36 @@ def populateSettingValuesFromConfigMap(parsedConfig)
       puts "config::error::Exception while reading config settings for cluster level environment variable collection - #{errorStr}, using defaults"
     end
   end
+
+  begin
+    if !parsedConfig.nil?  && !parsedConfig[:agent_settings][:health_model].nil? && !parsedConfig[:agent_settings][:health_model][:enabled].nil?
```

Comment:
could you add a nil check for agent_settings as well?

Reply:
It will be caught in the rescue and set to false if not present. I didnt want to add these checks, because for now it is a single value.